android-components/components/feature/customtabs/src/main/java/mozilla/components/feature/customtabs/CustomTabsToolbarFeature.kt

```diff
@@ -177,6 +177,7 @@ class CustomTabsToolbarFeature(
         if (config.menuItems.isNotEmpty() || menuBuilder?.items?.isNotEmpty() == true) {
             addMenuItems(config.menuItems, menuItemIndex)
         }
+        toolbar.display.hidePageActionSeparator()
```

Comment:
Could we move this closer to the top of the function? Something like after `toolbar.display.onUrlClicked = { false }`.
This is once again about trying to just hide the separator as soon as possible instead of doing it after adding all these action buttons and then suddenly the separator gap disappears and the buttons are shifting.

Reply:
I agree to this!